src/models/JobStatus.ts

```diff
@@ -1,5 +1,6 @@
 export enum JobStatus {
     Failed = 'failed',
     InProgress = 'in_progress',
-    Transcribed = 'transcribed'
+    Transcribed = 'transcribed',
+    Completed = 'completed'
```

Comment:
Should we consider creating a new enum for jobs statuses. Only old job types use `transcribed`

Reply:
I think we would just keep using the same enum but mark the `transcribed` field as deprecated eventually

Reply:
I guess one thing is that we have this CustomVocabularyStatus enum, but I don't think it makes sense for us to define new enums for every single service type